apps/portals/src/configurations/arkportal/style/_style_overrides.scss

```diff
@@ -47,20 +47,18 @@ td.SRC_noBorderTop:nth-child(n + 4) {
     }
   }
 }
-.bootstrap-4-backport {
-  .btn-secondary,
-  .btn-secondary-base {
-    color: white;
-    &:hover {
-      color: white;
-    }
-  }
-}
 
 .AboutPage {
   margin-top: 25px !important;
 }
 
+.MuiButtonBase-root.MuiButton-root.MuiButton-contained.MuiButton-containedSecondary {
+  color: white;
+  &:hover {
+    color: white;
+  }
+}
+
```

Comment:
Necessary to ensure that secondary button text in ARK is white -- otherwise, MUI will make the text black

Reply:
Oh, good call! Thanks for reminding me about the contrastThreshold -- it looks like we can get the button text color to stay white when the contrastThreshold is lowered to 2.3.